ksqldb-functional-tests/src/test/resources/query-validation-tests/suppress.json

```diff
@@ -20,9 +20,6 @@
     },
     {
       "name": "should not emit final result before window end if grace is specified as zero",
-      "properties": {
-        "ksql.suppress.enabled": true
```

Comment:
How come most of the tests have `"ksql.streams.__emit.interval.ms.kstreams.windowed.aggregation__": 0` added, but this one and the one immediately below it don't?

Reply:
Guess it still passed because we expect no output (and thus I missed it), but I think we should set it to zero to not accidentally mask a bug.

Great catch!